var/spack/repos/builtin/packages/netcdf-fortran/package.py

```diff
@@ -126,6 +132,12 @@ def configure_args(self):
                 # configuration failure, we set the following cache variable:
                 config_args.append("ac_cv_func_MPI_File_open=yes")
 
+        if "~shared" in netcdf_c_spec:
+            nc_config = which("nc-config")
+            config_args.append("LIBS={0}".format(nc_config("--libs", output=str)))
```

Comment:
Is this safe without `.strip()` to remove the trailing newline? I've had issues with others packages in this regard but it probably varies by build system.

Reply:
The trailing newline is removed somewhere inside the configure script, during the generation of `config.status`. But I agree that we should not rely on that.